WebContent/WEB-INF/jsp/users.jsp

```diff
@@ -416,6 +425,18 @@
               <td class="formLabelRequired"><fmt:message key="users.receiveOwnAuditEvents"/></td>
               <td class="formField"><input id="receiveOwnAuditEvents" type="checkbox"/></td>
             </tr>
+            <tr id="hideMenuRow" style="display:none;">
```

Comment:
reemplace style="display:none" by class="hidden"

and add theese to the most global stylesheet possible

.hidden { display:none; }
.hidden-important { display:none!important; }